src/main/java/zd/zero/waifu/motivator/plugin/assets/VisualMotivationAssetProvider.kt

```diff
@@ -53,13 +55,33 @@ object VisualMotivationAssetProvider {
             .flatMap { createAssetByCategory(it.random(random)) }
 
     private fun pickRandomAssetByCategory(category: WaifuAssetCategory): Optional<MotivationAsset> =
-        allOf(
-            TextAssetService.pickRandomAssetByCategory(category),
-            VisualAssetDefinitionService.getRandomAssetByCategory(category),
-            AudibleAssetDefinitionService.getRandomAssetByCategory(category)
-        ).map {
-            constructMotivation(it.first, it.second, it.third)
-        }
+        VisualAssetDefinitionService.getRandomAssetByCategory(category)
```

Comment:
I figured it was easiest to pick out an image then attempt to get the related assets from that. Considering the image pool is wayyyyy bigger than the other asset pools at the moment.